src/cpp/benders/benders_core/include/BendersStructsDatas.h

```diff
@@ -47,6 +47,7 @@ class WorkerMasterData {
   int _deleted_cut;
   int _nbasis;
   double _time;
+  double _subproblem_duration;
```

Comment:
Rename `_time` to `_master_time` to clarify ?

Reply:
_time is renamed _master_duration